agent/src/iconfucius/cli/chat.py

```
@@ -1322,136 +1328,161 @@ def run_chat(persona_name: str, bot_name: str, verbose: bool = False,
+    try:
+        if mcp_task is not None:
+            try:
+                from mcp.client.session import ClientSession
+                from mcp.client.streamable_http import streamablehttp_client
+
+                _mcp_ctx = streamablehttp_client(mcp_url)
+                read, write, _ = await _mcp_ctx.__aenter__()
+                _mcp_session_ctx = ClientSession(read, write)
+                mcp_session = await _mcp_session_ctx.__aenter__()
+                await mcp_session.initialize()
+            except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)